### PR TITLE
[docs] Add note about YCQL requiring Yugabyte-specific client drivers

### PR DESCRIPTION
**Pull request by def-**
Was recently noticed two times in QA, and there are existing bugs about this: https://github.com/yugabyte/yugabyte-db/issues/520
So there seems to be some confusion on this topic, best to clear that up in documentation.

### CODE REVIEW

File: docs/content/preview/reference/drivers/ycql-client-drivers.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The following client drivers are supported for use with the [Yugabyte Cloud Quer
 
 For tutorials on building a sample application with the following client drivers, click the relevant link included below for each driver.
 
+Note that for YCQL you have to use the Yugabyte-specific client drivers, otherwise you can run into `NotSupportedError: Unknown data type code 0x0080. Have to skip entire result set.` errors.
```
Comment (Arjun-yb) [inline docs/content/preview/reference/drivers/ycql-client-drivers.md:19]: this issue is not with most of the datatypes, I guess it is only with JSONB type, it is good to specify it clearly.

Comment (vijayjoshi16) [reply docs/content/preview/reference/drivers/ycql-client-drivers.md:19]: Yes, for the JSONB datatype the original `cassandra` driver does not support select query. But it is supported in `yb-cassandra-driver`. @def- and I were discussing about it a few days ago and we had to remove the existing installation of the original `cassandra` driver and install the `yb-cassandra-driver` again to get rid of this issue.

Comment (def-) [reply docs/content/preview/reference/drivers/ycql-client-drivers.md:19]: As discussed with Oleg, there are also other differences, can be slower or work incorrectly. Jsonb might just be the most obvious failure. So I think we should keep the recommendation general.

**Review by Arjun-yb — APPROVED** [review]

File: docs/content/preview/reference/drivers/ycql-client-drivers.md (hunk 1 above)
Comment (polarweasel) [inline docs/content/preview/reference/drivers/ycql-client-drivers.md:19]: Let's make this more generic and more prominent. How's this?
```suggestion
{{< note title="Use YugabyteDB client drivers" >}}
You should always use the YugabyteDB YCQL client drivers. Using generic Cassandra drivers can lead to errors and performance issues.
{{< /note >}}
```

**Review by polarweasel — APPROVED** [review]
Good as-is, better with my suggestion :)

**Pull request merged**